### DISCUSSION TITLE
My country not mentioned  in Two-factor authentication via SMS (Azerbaijan(+994)

### DISCUSSION THREAD

**Discussion by tofigf** [community/community#125448] [Code Security]
### Select Topic Area

General

### Body

Hi, I'm from Azerbaijan. I want to add  Two-factor authentication an additional layer of security to my account via SMS/Text message
on my GitHub profile and I tried to do so. Then when I tried to create SMS my country was not mentioned. Please consider adding my country code.
My country is Azerbaijan(+994)

**Comment by kazmatw** [comment]
Hello,

Thank you for reaching out with your concern about adding two-factor authentication via SMS to your GitHub profile. I understand the importance of having robust security measures, especially for crucial accounts like GitHub.

Currently, GitHub might not support SMS-based two-factor authentication for all country codes, including Azerbaijan (+994). Here are a few steps you can take:

Alternative Authentication Methods: Consider using other forms of two-factor authentication that GitHub supports, such as authentication apps (like Google Authenticator or Authy) or a security key. These methods are often more secure than SMS.

Contact GitHub Support: I recommend reaching out directly to GitHub support to request the addition of Azerbaijan to their list of supported countries for SMS authentication. You can do this through their official support page.

Stay Updated: Keep an eye on GitHub's updates or announcements. They continuously update their features, and your feedback might lead to a change in the future.

Community Feedback: Consider voicing this request on GitHub's community forums or on their feedback channels to see if other users are experiencing similar issues. There's strength in numbers, and a collective voice might expedite support for your request.

If you find this information useful, please consider selecting this as the answer to help others in the community identify effective solutions. Thank you!

**Comment by Boffyz** [comment]
still no authentication for Azerbaijan 🥲
